test/CodeToRest_test.py

```diff
@@ -636,11 +649,605 @@ def test_35(self):
+    def test_41(self):
+        self.mt('?- write("Hello world!").\n'
+                '% Comment here\n',
+                bf +
+                ' ?- write("Hello world!").\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Prolog'])
+
+    def test_42(self):
+        self.mt('/*\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '*/\n',
+                sl(-3) +
+                '\nmulti-\n'
+                'line\n'
+                'comment\n\n', ['Prolog'])
+
+    # AutoIt.
+    def test_43(self):
+        self.mt('ConsoleWrite(\'Hello World!\')\n'
+                '; Comment here\n',
+                bf +
+                ' ConsoleWrite(\'Hello World!\')\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['AutoIt'])
+
+    def test_44(self):
+        self.mt('#cs\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '#ce\n',
+                sl(-3) +
+                '\nmulti-\n'
+                'line\n'
+                'comment\n\n', ['AutoIt'])
+
+
+    # Perl.
+    def test_45(self):
+        self.mt('print "Hello World!"\n'
+                '# Comment here\n',
+                bf +
+                ' print "Hello World!"\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Perl'])
+
+    def test_46(self):
+        self.mt('print "Hello World!"\n',
+                bf +
+                ' print "Hello World!"\n' +
+                ef, ['Perl'])
+
+    def test_47_a(self):
+        self.mt('=pod\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '=cut\n',
+                sl(-3) +
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '\n', ['Perl'])
+
+    def test_47_b(self):
+        self.mt('=for\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '=cut\n',
+                sl(-3) +
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '\n', ['Perl'])
+
+    def test_47_c(self):
+        self.mt('=begin\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '=cut\n',
+                sl(-3) +
+                '\n'
+                'multi-\n'
+                'line\n'
+                'comment\n'
+                '\n', ['Perl'])
+
+    # Perl6.
+    """
+    def test_48_a(self):
+        self.mt('# Comment here\n',
+                sl(-3) +
+                'Comment here\n', ['Perl6'])
+
+    def test_48_b(self):
+        self.mt('say "Hello World!";\n'
+                '# Comment here\n',
+                bf +
+                ' say "Hello World!";\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Perl6'])
+
+    def test_49_a(self):

```

Comment:
I don't think this is correct Perl6 syntax -- see 1.7.2 at http://perl6intro.com/#_syntax_overview. Would you revise?